nemo/collections/asr/data/audio_to_label.py

```diff
@@ -202,6 +247,140 @@ def _vad_frame_seq_collate_fn(self, batch):
     return audio_signal, audio_lengths, tokens, tokens_lengths
 
 
+def _extract_seg_info_from_rttm(self, uniq_id, rttm_lines, tup_spks=None):
+    """
+    Get RTTM lines containing speaker labels, start time and end time. tup_spks contains two targeted
+    speaker indices for creating the groundtruth label file. Only speakers in tup_spks variable will be
+    included in the output lists.
+
+    Args:
+        uniq_id (str):
+            Unique file ID that refers to an input audio file and corresponding RTTM (Annotation) file.
+        rttm_lines (list):
+            List containing RTTM lines in str format.
+
+    Returns:
+        rttm_tup (tuple):
+            Tuple containing lists of start time, end time and speaker labels.
+
+    """
+    stt_list, end_list, speaker_list = [], [], []
+    if tup_spks:
+        label_scale_idx = max(self.emb_dict.keys())
+        mapping_dict = self.emb_dict[label_scale_idx][uniq_id]['mapping']
+        inv_map = {v: k for k, v in mapping_dict.items()}
+        bi_ch_infer_spks = []
+
+        for spk_idx in tup_spks[0]:
```

Comment:
is tup_spks a tuple? why are you looking at only tup_spks[0]

Reply:
changed the name of variables in collection to make it less confusing. tup_spks -> target_spks (self explanatory names)